lib/src/package_analysis/shape.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:json_annotation/json_annotation.dart';
+
+part 'shape.g.dart';
+
+@JsonSerializable()
+class PackageShape {
+  final List<LibraryShape> libraries;
+  final List<ClassShape> classes;
+
+  PackageShape(this.libraries, this.classes);
+
+  factory PackageShape.fromJson(Map<String, dynamic> json) =>
+      _$PackageShapeFromJson(json);
+
+  Map<String, dynamic> toJson() => _$PackageShapeToJson(this);
+}
+
+@JsonSerializable()
+class LibraryShape {
+  final String identifier;
```

Comment:
Should we call this `path`, `importPath` or `filepath` or? maybe `path` is fine?

Reply:
can `package:my_package/library.dart` really be called a path? I specifically didn't call this `path` because it's NOT the filesystem path for this library file

Reply:
uri?